packages/react-components/src/dialog/dialog.tsx

```diff
@@ -114,13 +114,14 @@ const StyledContent = styled(DialogPrimitive.Content, {
 
 type ContentProps = {
   children?: ReactNode;
+  overlayClassName?:string;
 } & ComponentProps<typeof StyledContent>;
 
-const Content = ({ children, ...props }: ContentProps) => {
+const Content = ({ children, overlayClassName, ...props }: ContentProps) => {
   const dialogContext = useContext(DialogContext);
   return (
     <DialogPrimitive.Portal>
-      <StyledOverlay />
+      <StyledOverlay {...(overlayClassName && { className: overlayClassName })} />
```

Comment:
why not expose a classname from twigs and you can use it however you want? instead of doing the other way?

Reply:
There are certain cases where we can have multiple dialogs open at a time and need to style a specific overlay. In that case wouldn't it be better to keep a standard class as static and then add support for custom class name?